docs/requirements.txt

```diff
@@ -1,6 +1,6 @@
 # pin docutils at 0.15.2 to avoid transitive dependency conflict with botocore (requires < 0.16)
 docutils==0.15.2
-pillow
+pillow<10.0.0
```

Comment:
What does the pin here do? Since `torch` isn't included in docs (pinned or not), I don't think it's necessary.